tools/hazelcast.xml

```diff
@@ -1,13 +1,17 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config hazelcast-basic.xsd"
-           xmlns="http://www.hazelcast.com/schema/config"
-           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<hazelcast xmlns="http://www.hazelcast.com/schema/config"
```

Comment:
Can the hazelcast XML config be removed altogether?

Reply:
Yes it is possble but it is very easy way to add new peroperties in hazelcast cluster, with java config we need a new build. I would like to keep this setup for in house deployment to quickly change and test properties

Reply:
OK, but lets not forget to get rid of it. I'll create a separate task for this.